src/main/java/bio/terra/service/dataset/DatasetTable.java

```diff
@@ -105,4 +106,9 @@ public DatasetTable setRowCount(Long rowCount) {
     this.rowCount = rowCount;
     return this;
   }
+
+  @Override
+  public String toLogString() {
+    return String.format("%s (%s)", this.getName(), this.getId());
```

Comment:
Why do we need the this.get() methods here? 
```suggestion
    return String.format("%s (%s)", name, id);
```

Reply:
I typically prefer going through getters as a force of habit